services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/statistics/StatisticsService.java

```diff
@@ -108,9 +108,23 @@ public void startUp(FloodlightModuleContext context) {
         if (interval > 0) {
             threadPoolService.getScheduledExecutor().scheduleAtFixedRate(
                     () -> switchService.getAllSwitchMap().values().forEach(iofSwitch -> {
-                        gatherPortStats(iofSwitch);
-                        gatherFlowStats(iofSwitch);
-                        gatherMeterStats(iofSwitch);
+                        try {
+                            gatherPortStats(iofSwitch);
+                        } catch (Exception e) {
+                            logger.error("Failed to gather stats for ports on switch {}.", iofSwitch.getId(), e);
```

Comment:
2 issue - first it will not provide traceback (you can't use "{}" formatting and pass exception object, you must use String.format to format message if you pass exception object and expect traceback in log).

And second - make different log messages for different sections.

Reply:
1. will fix.
2. messages are different